app/home/factory/login-factory.coffee

```diff
@@ -22,19 +20,19 @@ angular
         @pswSession = ""
         for aux, i in users
           if user is aux.user and psw is aux.psw
-            alert user
-            alert psw
+            name = aux.name
+            surname = aux.surname
             @loginSuccess = true
+            #Save data locally
             @userSession = localStorageService.set('user', @user)
             @pswSession = localStorageService.set('psw', @psw)
-            
-        if @loginSuccess is true
-          alert "here I need a message: Welcome + name, it's correct"
-          location.href = '/home/home.tpl.html'
-        else
-          alert "here I need a message: it's failed"
+            @nameSession = localStorageService.set('name', name)
+            @surnameSessin = localStorageService.set('surname', surname)
```

Comment:
This version of login-factory.coffe is old, in the new one, location.href not exits.

In regard to the units test, now I start with it, firstly i'm going to read a bit about it.